python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_swarm_group_chat.py

```diff
@@ -197,14 +205,18 @@ async def main() -> None:
     def __init__(
         self,
         participants: List[ChatAgent],
+        *,
```

Comment:
Potentially a breaking change, but it's small enough that I think it is likely okay. Probably should have been this way from the start anyway

Reply:
Yeah... It should have been this way. I am planning to make it clear in the release note that we require keyword arguments for these classes.

Reply:
I reverted this ... let's wait until a good time.